src/slider.js

```diff
@@ -515,8 +515,9 @@ function slider(orientation, scale) {
 
   function updateValue(newValue, notifyListener) {
     if (
-      value[0] !== newValue[0] ||
-      (value.length > 1 && value[1] !== newValue[1])
+      value.some(function (d, i) {
+        return d !== newValue[i];
```

Comment:
This isn't correct if `newValue` is shorter than `value`. Perhaps we could check if the arrays are the same length first? If they differ in length then we skip the check and always update.